docs/tutorials/fairplay.md

```diff
@@ -76,3 +76,24 @@ player.getNetworkingEngine().registerResponseFilter((type, response) => {
   response.data = shaka.util.Uint8ArrayUtils.fromBase64(responseText).buffer;
 });
 ```
+
+## DRM Engine
+
+Some FairPlay support needs to be detected in order to change source to HLS if FairPlay is supported (AFAIK currently FairPlay is supported only within HLS and not publicly supported in DASH). You can query EME yourself, directly, and only query FairPlay, rather than the full list of key systems probed by `DrmEngine`. 
+
+```js
+async function isFairPlaySupported() {
```

Comment:
It may be a good idea to add this to `shaka.util.FairPlayUtils` so apps can just call this function.

Reply:
Should I go ahead and add it to [fairplay_utils.js](https://github.com/google/shaka-player/blob/master/lib/util/fairplay_utils.js)?